modules/db/src/main/java/org/mule/module/db/internal/domain/connection/DefaultDbConnection.java

```diff
@@ -34,6 +43,9 @@
  */
 public class DefaultDbConnection extends AbstractDbConnection
 {
+    protected transient Log logger = LogFactory.getLog(getClass());
+
+    public static final int ATTR_TYPE_NAME_INDEX = 6;
```

Comment:
is this constant among db engines? if so, is there no constant or an alternative way to get it in the JDBC apis?

Reply:
It is the jdbc api

https://docs.oracle.com/javase/8/docs/api/java/sql/DatabaseMetaData.html#getAttributes-java.lang.String-java.lang.String-java.lang.String-java.lang.String-

https://docs.oracle.com/javase/7/docs/api/java/sql/DatabaseMetaData.html

But there is no constant in the jdbc api for that index, that I find. Every example I found uses the index without a constant (in a sense I find it reasonable as it is something related exclusively to the method).